tests/model/utils/selection-post-fixer.js

```diff
@@ -907,7 +954,7 @@ describe( 'Selection post-fixer', () => {
 				expect( getModelData( model ) ).to.equal( '<paragraph>foob[a]r</paragraph>' );
 			} );
 
-			it( 'should not fix #2', () => {
+			it( 'should not fix #2 (inline widget selected)', () => {
```

Comment:
Consistency: you use parenthesis here. In other tests you use a dash.

Reply:
I've used parenthesis everywhere as it was used more times then dash.